contracts/tests/MessageProcessor.test.ts

```diff
@@ -34,7 +35,7 @@ import { timeTravel, deployTestContracts } from "./utils";
 describe("MessageProcessor", () => {
   // contracts
   let maciContract: MACI;
-  let pollContract: PollContract;
+  // let pollContract: PollContract;
```

Comment:
let's remove commented out code

Reply:
there's more instances in this file, please let's remove everywhere

Reply:
resolved